common/models.py

```diff
@@ -29,26 +29,7 @@ class EmailVerificationTable(models.Model):
     token = models.CharField(max_length=100)
 
     def send_verification_email(self):
-        html_message = render_to_string(
-            "registration/verify_email.html",
-            {
-                "token": self.token,
-            },
-        )
-        send_mail(
-            "Verify your email address on SciLifeLab Serve",
-            (
-                f"You registered an account on SciLifeLab Serve ({DOMAIN}).\n"
-                "Please click this link to verify your email address:"
-                f" https://{DOMAIN}/verify/?token={self.token}"
-                "\n\n"
-                "SciLifeLab Serve team"
-            ),
-            settings.EMAIL_HOST_USER,
-            [self.user.email],
-            fail_silently=False,
-            html_message=html_message,
-        )
+        send_verification_email(self.user.email, self.token)
```

Comment:
Maybe we should change the name here or somehow distinguish the two functions? Looks a bit odd imo:)

Reply:
That's a good idea